drivers/gpu/drm/panel/panel-xiaomi-f4-41-06-0a.c

```diff
@@ -300,7 +300,8 @@ static void xiaomi_f4_41_06_0a_remove(struct mipi_dsi_device *dsi)
 }
 
 static const struct of_device_id xiaomi_f4_41_06_0a_of_match[] = {
-	{ .compatible = "xiaomi,f4-41-06-0a" }, // FIXME
+	{ .compatible = "xiaomi,f4-41-06-0a" },
+	{ .compatible = "visionox,g2647fb105" },
```

Comment:
Let's just rename the panel driver entirely here if we know that this is the panel